lib/msal-common/src/account/AuthToken.ts

```diff
@@ -47,4 +47,27 @@ export class AuthToken {
             throw ClientAuthError.createTokenParsingError(err);
         }
     }
+
+    /**
+     * Determine if the token's max_age has transpired
+     */
+    static checkMaxAge(authTime: number | undefined, maxAge: number): void {
+        /*
+         * per https://auth0.com/docs/authenticate/login/max-age-reauthentication
```

Comment:
Please link to the official RFC for this and not another vendor's documentation.

Reply:
max_age is not mentioned in the official rfc [here](https://www.ietf.org/rfc/rfc5849.txt), however it is mentioned in the official OIDC spec [here](https://openid.net/specs/openid-connect-core-1_0.html#AuthRequest). I will link to the OIDC spec.